website/docs/r/virtual_machine_extension.html.markdown

```diff
@@ -69,10 +71,10 @@ resource "azurerm_storage_container" "test" {
 
 resource "azurerm_virtual_machine" "test" {
   name                  = "acctvm"
-  location              = "West US"
+  location              = "${azurerm_resource_group.test.location}"
   resource_group_name   = "${azurerm_resource_group.test.name}"
   network_interface_ids = ["${azurerm_network_interface.test.id}"]
-  vm_size               = "Standard_A0"
+  vm_size               = "Standard_F2"
```

Comment:
This VM type is not available in all regions.

Reply:
that's fine as A0 isn't available in some of the newer regions iirc - really we should stop using A0's in examples/tests since they take considerably longer to boot than other machines (since they're 0.25 cores)